CONTRIBUTING.md

```diff
@@ -1,13 +1,26 @@
 # How to Contribute
 
-One of the best ways to contribute is to participate in discussions and discuss issues. You can also contribute by submitting pull requests with code changes.
+One of the best ways to contribute is to participate in discussions and discuss issues. Currently submitting pull requests is diabled for users that do not have write access.
 
 ## API Design Guidelines
 
 Please review the [architectural guidelines](https://github.com/OfficeDev/microsoft-teams-library-js/wiki/Library-Architecture) when adding/changing functionality in the teams-js library.
 
 ## General Contribution Guide
 
+1. Unless it is a trivial change, make sure that there is a corresponding issue for your change first. If there is none, create one.
+2. If the issue is trivial please refer to the [Legacy Contribution Guide](#legacy-contribution-guide)
+
```

Comment:
I'm a little confused about why we're directing developers to follow the Legacy Contribution Guide if we've disabled PRs from forks.

Reply:
You're right, this is confusing. I wanted to allow the user to still be able to work on a fork if they'd like, I've replaced this with a new `Contribution Guide for Users without Write Access section`